cfme/tests/infrastructure/test_vm_migrate.py

```diff
@@ -32,6 +32,7 @@ def new_vm(setup_provider_modscope, provider, request):
     return vm
 
 
+@pytest.mark.rhv1
```

Comment:
I am not saying this is wrong but we could use discussion about this as well

Reply:
OK. VM migration is a major feature in RHV - This is why it is marked as T1.